docs/toc.yml

```diff
@@ -7,13 +7,13 @@
 
 - name: Get started
   items:
-  - name: Start Here! A guide for beginners
+  - name: A guide for beginners
     href: overview/learning-path-beginner.md
     displayName: develop
-  - name: Transition Here! A guide for VSTO add-in developers
+  - name: A guide for VSTO add-in developers
     href: overview/learning-path-transition.md
     displayName: develop
-  - name: Building Office Add-ins
+  - name: Office Add-ins development process
```

Comment:
Maybe...
```suggestion
  - name: Development process overview
```

Reply:
I like this (and your next suggestion!). The two updates together have a helpful consistency in term usage.